### DISCUSSION TITLE
Data lost when transferring issues between repositories

### DISCUSSION THREAD

**Discussion by nicksloan** [community/community#88557] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

When I transfer an issue from one repository in my organization to another, occasionally associated data is lost along the way. This includes assignments, labels, and project properties. This kind of data loss slows down our work and makes issue management unreliable. It also appears to be unrecoverable, because the missing data no longer appears in the issue timeline.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by jackbentley** [comment]
I've noticed when transferring issues that anything in the timeline relating to a Project disappears. The new issue is assigned to the project with the correct status, though. So, this is transferred properly just the timeline info disappears.

**Comment by jdhenry08** [comment]
I'm seeing something similar - When I transfer an issue from one repo to another, it unlinks from the project and any project-related fields are lost.

EDIT: To clarify, this seems to only happen if the linked project is in a different org than the issue.
